tasks/managed/push-snapshot/push-snapshot.yaml

```diff
@@ -348,10 +348,17 @@ spec:
             exit 1
         fi
 
-        PUSHES=$(jq -s . "$TMP_RESULTS_DIR"/*.json)
-        jq --argjson PUSHES "$PUSHES" '
-          reduce $PUSHES[] as $p (.; (.images[] | select(.name == $p.name).urls) += [$p.url])
+        # Create a temporary file for the pushes data to avoid command line argument length limits
+        PUSHES_FILE=$(mktemp)
+        jq -s . "$TMP_RESULTS_DIR"/*.json > "$PUSHES_FILE"
+        
+        # Use file input instead of command line arguments to avoid argument length limits
+        jq --slurpfile PUSHES "$PUSHES_FILE" '
+          reduce $PUSHES[0][] as $p (.; (.images[] | select(.name == $p.name).urls) += [$p.url])
         ' "$RESULTS_JSON_FILE" | tee "$RESULTS_FILE"
+        
+        # Clean up temporary pushes file
+        rm -f "$PUSHES_FILE"
```

Comment:
nit - you could just add this to the list 2 lines below and change the comment to `s/file/files/` (which should already be the case)

Reply:
fixed